README.md

```diff
@@ -8,6 +8,47 @@ The `CRUD` module allows to perform CRUD operations on the cluster.
 It also provides the `crud-storage` role for
 [Tarantool Cartridge](https://github.com/tarantool/cartridge).
 
+# Supported Tarantool versions
```

Comment:
The idea to describe supported versions appears when we discussed [this](https://github.com/tarantool/crud/pull/198) change. I had the question about crud.min() / crud.max(): on which tarantool versions the module has no support of those function. This information is not present here.

Reply:
Just for context: commits that remove `crud.mix()`/`crud.max()` support in previous version:
- https://github.com/tarantool/crud/commit/5edf0274f28a2e607d0f34fa3423d00ce1d44f14
- https://github.com/tarantool/crud/commit/56247f27239e1f2e92701c9ad159e3977b512a79

Reply:
These commit reverted in https://github.com/tarantool/crud/pull/215

Reply:
The initial questions was "What is a minimal working version of Tarantool for CRUD?". I have prepared a table with information (for us) about Tarantool features used by CRUD in Tarantool versions and additionally a list with Tarantool versions starting from user can use CRUD.